File: Célia_LingProgram/ListaJavaFacul/src/L3_Exer07_Acumulador/Exer07_Acumulador.java
package L3_Exer07_Acumulador;

import java.util.Random;

public class Exer07_Acumulador {
    public static void main(String[] args) {
        Integer sorteio;
        Integer a = 0;

        Random aleatorio = new Random();

        do {
            sorteio = aleatorio.nextInt(6);
            a += sorteio;
            System.out.println("Número: " + sorteio);
//          System.out.println("Total:" +  a);
        } while (sorteio != 0);

        System.out.println("Total: " + a);

    }
}

File: Célia_LingProgram/ListaJavaFacul /src/L3_Exer07_Acumulador/Exer07_Acumulador.kt
package Lista03

import kotlin.random.Random

//
//Criar uma classe chamada “Acumulador” que :
//Leia vários números até ler o número 0 e calcule a soma dos números lidos. (Não é para usar vetor)

fun main() {

    var sorteio = 4
    var a = 0

    do{
        val random = Random.nextInt(0, 6)
        a += sorteio
        println("Número: " + sorteio)

    }while (sorteio != 0)

    println("Total: " + a)
}